### PR TITLE
change 1 only on branch1

### PR DESCRIPTION
**Pull request by jltabc**
jltabcGmail - see the change I made. merge if like it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# firstrepo
\ No newline at end of file
+# firstrepo
+# change 1 only on branch1
```
Comment (jltabcGmail) [inline README.md]: say instead "change 1 by jltabc (and not by jltabcGmail)only on branch1"

Comment (jltabc) [reply README.md]: taken action

**Review by jltabcGmail — APPROVED** [review]
approved and merged

**Comment by jltabcGmail** [comment]
merging changes suggested by jltabcGmail

**Pull request merged**